predicates:
  at:
    max_distance: 1.0
  on-pred:
    max_above: 0.03
explorer:
  search_budget_sec: 700
  alternating_sequence_length: false
  min_sequence_length: 4
  max_sequence_length: 4
  max_failed_samples: 50
  bounding_box_inflation_length: 0.1
  action_denylist: ["nav-in-reach", "nav-at", "grasp"]
  predicate_precondition_allowlist: ["inside", "on"]
  radii: [0.1, 1.5, 10.0]
  distance_limit_closeby_objects: 0.25
robot_arm:
  max_force_magnitude: 150
grasping:
  pregrasp_z_offset: 0.15
mcts:
  search_budget_sec: 700
  max_depth: 12
